Library/Formula/libswiften.rb

```diff
@@ -1,24 +1,13 @@
 class Libswiften < Formula
   desc "C++ library for implementing XMPP applications"
   homepage "https://swift.im/swiften"
-  revision 2
 
   stable do
-    url "https://swift.im/downloads/releases/swift-2.0/swift-2.0.tar.gz"
-    sha256 "cbcdbe527dc4d112a38f3cdec5f1051d3beba0b97f8082f90debd04a5b45a41f"
+    url "git://swift.im/swift", :tag => "swift-3.0", :revision => "14ddf8b470f5a3420b5f2c96daea33c2513cac6e"
```

Comment:
I don't *think* this is true. I downloaded the release tarball (https://swift.im/downloads/releases/swift-3.0/swift-3.0.tar.gz), compiled it, and in Version.h I see
`
#pragma once

#define SWIFTEN_VERSION 0x030000
#define SWIFTEN_VERSION_STRING "3.0"
`